google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.java

```diff
@@ -346,6 +346,22 @@ public Builder setCompressorName(String compressorName) {
       return this;
     }
 
+    /**
+     * Sets the default missing value interpretation value if the column is not presented in the
+     * missing_value_interpretations map.
+     *
+     * <p>If this value is set to `DEFAULT_VALUE`, we will always populate default value if the
+     * field is missing from json and default value is defined in the column.
+     *
+     * <p>If this value is set to `NULL_VALUE`, we will always not populate default value.
```

Comment:
Change the comment of 'NULL_VALUE' to: we will always populate NULL to the missing fields, which is currently our default behavior.